### PR TITLE
Correctly remove split/merged packages in PesEventsScanner + unit test

### PR DESCRIPTION
**Pull request by drehak**
Fixes https://github.com/oamg/leapp-repository/issues/177.
Recreating since I accidentally pushed my branch to the main repo instead of my fork.

### CODE REVIEW

**Comment by drehak** [comment]
e2e tests

**Comment by vojtechsokol** [comment]
leapp-ci build

**Comment by vojtechsokol** [comment]
e2e tests

File: repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py [hunk 1]
```diff
@@ -286,6 +287,27 @@ def report_skipped_packages(message, packages):
...
+    Output packages from an event are added to packages for removal only if all input packages are already there.
+
+    :param events: List of Event tuples, where each event contains event type and input/output pkgs
+    :param transaction_configuration: RpmTransactionTasks model instance with pkgs to install, keep and remove based
+                                      on the user configuration files
+    """
+    message = 'Marking packages for removal as a result of events:\n'
+
+    for event in events:
+        if event.action in ('Split', 'Merged'):
```
Comment (vojtechsokol) [inline repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py]: What about `Replaced` and `Renamed` events?

Comment (drehak) [reply repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py]: After discussing this with others, it makes sense to apply this PR to `Replaced` and `Renamed` too. I'll add those, thanks for the remark.

**Comment by vojtechsokol** [comment]
Please rebase to master, CI issues should be fixed by now.

**Comment by drehak** [comment]
leapp-ci build

**Comment by vojtechsokol** [comment]
One more rebase is needed due to #217.

**Comment by drehak** [comment]
Still not 100% confident with force pushes, please let me know if I messed up something.

**Pull request merged**